packages/storybook/stories/inbox/custom-components.tsx

```diff
@@ -23,3 +23,7 @@ export const CustomTabs: React.FunctionComponent = () => {
 export const CustomNoMessages: React.FunctionComponent = () => {
   return <div>No Messages</div>;
 };
+
+export const CustomBell: React.FunctionComponent = () => {
+  return <div>Bell</div>;
```

Comment:
can this verify the props?

Reply:
Not sure how we can do it here? I console logged in local and could see it -
<img width="770" alt="Screen Shot 2022-01-28 at 10 57 25 AM" src="https://user-images.githubusercontent.com/6154318/151605401-cc4f01ea-e43b-4b6b-a83e-17226553e4f1.png">